index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <title>Homepage</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="home.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="My Logo">
+        <img src="image.png" alt="">
+    </div>
```

Comment:
- [ ] Please make sure you change your `div` tag to a `header` tag to make your code semantic and more understandable for people with screen readers. You can read more about semantic HTML in this [article](https://www.w3schools.com/html/html5_semantic_elements.asp). 🙏 
- [ ] Please change the `image` tag to the actual contents 🙏 .